official/utils/logs/logger.py

```diff
@@ -391,24 +387,6 @@ def _collect_cpu_info(run_info):
         "'cpuinfo' not imported. CPU info will not be logged.")
 
 
-def _collect_gpu_info(run_info, session_config=None):
```

Comment:
This code path is not used anymore and it is broken if using CUDA_VISIBLE_DEVICES.  Any work to fix it is not useful because we use perfzero for logging.  The entire code path can be removed at some point, just not now.